docs/kubernetes/kctlr-f5-resource.rst

```diff
@@ -87,12 +97,12 @@ Example F5 Resources
 F5 Virtual Server ConfigMap
 ```````````````````````````
 
-The example below creates an HTTP virtual server with a pool, pool member, and health monitor for the Service named "myService". The |kctlr| will create the virtual server in the :code:`k8s` partition (must already exist on the BIG-IP system).
+The example below creates an HTTP virtual server with a pool, pool member, and health monitor for the Service named "myService".
+The |kctlr| will create the virtual server in the :code:`k8s` partition, which already exists on the BIG-IP system.
 
 .. _f5-resource-blob:
 
 .. literalinclude:: /kubernetes/config_examples/f5-resource-vs-example.configmap.yaml
```

Comment:
This ConfigMap now refers to mySuperAwesomeApp.

Reply:
Are you intending to link this: f5-resource-configmap-https.yaml?

Reply:
fixed